### ISSUE TITLE
Images Link Automatically

### ISSUE THREAD

**Issue by warmbowski** [hackmdio/hackmd#214]
I noticed that the image inlined with `![alt text](image-src)` get wrapped in a link tag with the src attrib as the href attrib. I this proper markdown behavior? There are two problems with this:
1. How is it overridden if you don't want it linked to the src. 
2. When printing, the link url is printed next to the image. 

Number two being the one that really bugs me. Hackmd.io is great because it prints out so nicely, as compared to many other MD editors, but for this problem of the link printing.

I've seen that you can add this code to disable the link print, but I don't know if you want to do that. 
```
@media print {
  a[href]:after {
    content: none !important;
  }
}
```

Maybe the best course is to not automatically linkify images and let the user add links to images if they want them.

**Comment by warmbowski** [comment]
My workaround is to wrap images in a link to `#`, which overrides the auto link.

`[![alt text](image-src)](#)`

Let me know if this auto linking of images is intended or not. Maybe I can figure out how to change if need be.

**Comment by jackycute** [comment]
Hi @warmbowski !
The linkify image is not markdown intent to do.
That is I made so to simulate GitHub behavior.
I think it links it because you could view full image when you click it, but this also can achieve by JS.
Since you bring it up, I will try to figure out and fix it.

Thanks!

**Comment by jackycute** [comment]
I think it's ok to remove auto linkify image since users can above reference syntax to link out.

**Comment by jackycute** [comment]
Fixed in 70a52e2eaed25686160f0d1be1682f85230d9408
